commit e14a91e59d35bf2fa649a9726ccce838b8c6e4b7
Author: Adhemerval Zanella Netto <adhemerval.zanella@linaro.org>
Date:   Wed Oct 26 16:04:25 2022 -0300

    time: Use 64 bit time on tzfile
    
    The tzfile_mtime is already compared to 64 bit time_t stat call.
    Reviewed-by: DJ Delorie <dj@redhat.com>
    
    (cherry picked from commit 4e21c2075193e406a92c0d1cb091a7c804fda4d9)

diff --git a/time/tzfile.c b/time/tzfile.c
index 8668392ad387af05..84fd9df65f76f148 100644
--- a/time/tzfile.c
+++ b/time/tzfile.c
@@ -32,7 +32,7 @@
 int __use_tzfile;
 static dev_t tzfile_dev;
 static ino64_t tzfile_ino;
-static time_t tzfile_mtime;
+static __time64_t tzfile_mtime;
 
 struct ttinfo
   {
